evm/vm/forks/sharding/transactions.py

```diff
@@ -43,5 +44,6 @@ def _get_sharding_intrinsic_gas(transaction_data, transaction_code):
     return (
         GAS_TX +
         num_zero_bytes * GAS_TXDATAZERO +
-        num_non_zero_bytes * GAS_TXDATANONZERO
+        num_non_zero_bytes * GAS_TXDATANONZERO +
+        GAS_TXCREATE
```

Comment:
I may miss something, could you tell me why `GAS_TXCREATE` is included in every sharding tx?

Reply:
You are right. It should not be included in every tx.